### DISCUSSION TITLE
alert details on "resolved" slack message

### DISCUSSION THREAD

**Discussion by matrog** [grafana/grafana#55086] [Alerting]
When i receive an email for a resolved alert I have all the details for that alert while for Slack I have details only for "Alerting" message while the "resolved" is empy so I don't know which alert is resolved.

As you can see from the picture the Alert message is well detail while the Resolved is empty

<img width="635" alt="Slack___cf-notifications-testalert___IBM_Cloud_Platform___2_new_items" src="https://user-images.githubusercontent.com/103855/189901642-35b665bd-ec3e-4258-8dd2-5ee23c5dc8d2.png">

**Comment by greatvovan** [comment]
Curious, have you tried to create your own template? Like this: https://grafana.com/docs/grafana/latest/alerting/contact-points/message-templating/example-template/ Slack integration does support templating for title and body, as I can see

<img width="566" alt="image" src="https://user-images.githubusercontent.com/4903007/193169505-611a18b9-2d4b-469f-ac70-c277709142e1.png">

**Comment by matrog** [comment]
already using template but the resolved message is still empty

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
